pkgs/servers/nextcloud/packages/nextcloud-apps.json

```diff
@@ -12,7 +12,6 @@
 , "gpoddersync": "agpl3Only"
 , "groupfolders": "agpl3Plus"
 , "impersonate": "agpl3Plus"
-, "keeweb": "agpl3Plus"
```

Comment:
keeweb seems to get updates again https://github.com/jhass/nextcloud-keeweb/releases
there are also pushed to apps.nextcloud.com

Reply:
The nextcloud app did get some updates, but the project itself did not: https://github.com/keeweb/keeweb/issues/2022

Reply:
ah i see okay